float64 tcp
float64 ori
int64 zone
float64 hand_speed
float64 hand_force
geometry_msgs/Pose pose